$NetBSD: patch-bk,v 1.3 2025/02/27 10:55:13 nia Exp $

This patch was originally added to fix builds with sunpro.

--- src/CurrentWindowCmd.hh.orig	2008-09-01 16:59:05.000000000 +0100
+++ src/CurrentWindowCmd.hh	2009-07-06 12:53:48.000000000 +0100
@@ -144,7 +144,7 @@
 // move cmd, relative position
 class MoveCmd: public WindowHelperCmd {
 public:
-    explicit MoveCmd(const int step_size_x, const int step_size_y);
+    explicit MoveCmd(int step_size_x, int step_size_y);
     static FbTk::Command<void> *parse(const std::string &command,
                                 const std::string &args, bool trusted);
 protected:
